core/src/main/java/greencity/repository/UserRepo.java

```diff
@@ -79,4 +79,37 @@
      */
     @Query("SELECT profilePicturePath FROM User WHERE id=:id")
     Optional<String> getProfilePicturePathByUserId(Long id);
+
+    /**
+     * Get all user friends{@link User}.
+     *
+     * @return list of {@link User}.
+     */
+    @Query(value = " SELECT * FROM public.users_friends "
```

Comment:
Why this query is written in native SQL? I would prefer JPQL. Reasoning: JPQL is independent of database server dialect. Event SQL is a standard each server (Postgres, Oracle, mySql) have dialects. The same bellow.